src/design/plone/ioprenoto/adapters/stringinterp.py

```diff
@@ -62,3 +63,24 @@ def safe_call(self):
             booking_folder.getPhysicalPath()[len(portal.getPhysicalPath()) :]  # noqa
         )
         return f"{portal_url}/{booking_folder_path}?tab=search&SearchableText={self.context.getBookingCode()}&login=1"
+
+
+@adapter(Interface)
+class BookingUnitaOrganizzativaTitle(BaseSubstitution):
+    def safe_call(self):
+        return getattr(
+            getattr(
```

Comment:
unreadable ... :)

Reply:
Is get attribute title of first object in uffici_correllati field. Do I need to refactor this or write a short comment?

Reply:
the code is unreadable.
Can you split it into a better human-readable way? Nested call functions makes my eyes bleed

Reply:
Done, review pls